--- a/usr/sbin/start_pms	2019-01-04 14:57:17.158769341 +0100
+++ b/usr/sbin/start_pms	2019-01-04 15:09:54.555128630 +0100
@@ -20,6 +20,9 @@
 
 test -f /etc/default/plexmediaserver && . /etc/default/plexmediaserver
 
+# Activate python virtualenv
+. "${PLEX_MEDIA_SERVER_HOME}"/Resources/Python/bin/activate
+
 if [ ! -d "$PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR" ]
 then
   mkdir -p "$PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR"
