diff --git a/CMakeModules/WebClientResources.cmake b/CMakeModules/WebClientResources.cmake
--- a/CMakeModules/WebClientResources.cmake
+++ b/CMakeModules/WebClientResources.cmake
@@ -9,18 +9,7 @@ if(NOT SKIP_WEB_CLIENT)
   message(STATUS "web-client version: ${WEB_CLIENT_VERSION}")
 
   file(
-    DOWNLOAD ${WEB_CLIENT_URL} ${CMAKE_CURRENT_BINARY_DIR}/${WEB_CLIENT_CPP}.tbz2
-    EXPECTED_HASH SHA1=${WEB_CLIENT_HASH}
-    TIMEOUT 100
-    SHOW_PROGRESS
-    TLS_VERIFY ON
-  )
-
-  add_custom_command(
-    OUTPUT ${WEB_CLIENT_CPP}
-    COMMAND ${CMAKE_COMMAND} -E tar xjf ${CMAKE_CURRENT_BINARY_DIR}/${WEB_CLIENT_CPP}.tbz2
-    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${WEB_CLIENT_CPP}.tbz2
-    COMMENT "Unpacking: ${WEB_CLIENT_CPP}.tbz2"
+    RENAME ${CMAKE_SOURCE_DIR}/plex-web-client-konvergo-${WEB_CLIENT_VERSION}.cpp ${CMAKE_CURRENT_BINARY_DIR}/${WEB_CLIENT_CPP}
   )
 
   add_custom_target(UnpackWebClientResource
