diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -54,7 +54,7 @@ from classes.logger import log
 log.info("Execution path: %s" % os.path.abspath(__file__))
 
 # Boolean: running as root?
-ROOT = os.geteuid() == 0
+ROOT = False
 # For Debian packaging it could be a fakeroot so reset flag to prevent execution of
 # system update services for Mime and Desktop registrations.
 # The debian/openshot.postinst script must do those.
