diff --git a/CMakeLists.txt b/CMakeLists.txt
index c4dc88b..2784f79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,6 +47,8 @@ set_property(TARGET ${LIBRARY_NAME} PROPERTY CXX_STANDARD_REQUIRED ON)
 add_executable(${EXECUTABLE_NAME} src/options.cpp
                                   src/main.cpp)
 
+set_property(TARGET ${EXECUTABLE_NAME} PROPERTY CXX_STANDARD 11)
+
 set( CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/modules" )
 
 find_package(GLM REQUIRED)
@@ -61,12 +63,13 @@ include_directories(${X11_INCLUDE_DIR}
                     ${GLX_INCLUDE_DIR}
                     ${OPENGL_INCLUDE_DIR})
 
-target_link_libraries(${EXECUTABLE_NAME} ${X11_LIBRARIES}
-                                         ${GLM_LIBRARIES}
-                                         ${OPENGL_LIBRARIES}
-                                         ${GLX_LIBRARY}
-                                         ${XEXT_LIBRARIES}
-                                         ${LIBRARY_NAME})
+target_link_libraries(${LIBRARY_NAME} ${X11_LIBRARIES}
+                                      ${GLM_LIBRARIES}
+                                      ${OPENGL_LIBRARIES}
+                                      ${GLX_LIBRARY}
+                                      ${XEXT_LIBRARIES})
+
+target_link_libraries(${EXECUTABLE_NAME} ${LIBRARY_NAME})
 
 install( TARGETS ${EXECUTABLE_NAME} DESTINATION "${CMAKE_INSTALL_PREFIX}/bin" )
 install( TARGETS ${LIBRARY_NAME} DESTINATION "${CMAKE_INSTALL_PREFIX}/lib" )
diff --git a/src/slop.cpp b/src/slop.cpp
index b149b78..b6b472a 100644
--- a/src/slop.cpp
+++ b/src/slop.cpp
@@ -1,6 +1,8 @@
 #include <chrono>
 #include <thread>
 
+#include <stdlib.h>
+
 #include "slopstates.hpp"
 #include "mouse.hpp"
 #include "resource.hpp"
@@ -36,11 +38,16 @@ slop::SlopOptions::SlopOptions() {
     padding = 0;
     shader = "textured";
     highlight = false;
-    xdisplay = ":0";
     r = 0.5;
     g = 0.5;
     b = 0.5;
     a = 1;
+
+    char * envdisplay = getenv("DISPLAY");
+    if (envdisplay == NULL)
+        xdisplay = ":0";
+    else
+        xdisplay = envdisplay;
 }
 
 slop::SlopSelection::SlopSelection( float x, float y, float w, float h, int id ) {
