diff --git a/smd-applet.vala b/smd-applet.vala
index fca16d6..3d9d85d 100644
--- a/smd-applet.vala
+++ b/smd-applet.vala
@@ -344,7 +344,8 @@ class smdApplet : Gtk.Application {
 		});
 
 		// notification system
-		unowned List<string> l = Notify.get_server_caps();
+		//unowned List<string> l = Notify.get_server_caps();
+		var l = Notify.get_server_caps();
 		notification_server_has_persistence = (0 <= l.index("persistence")); 
 
 		// error mode data
