diff --git plex-home-theater-1.3.5/plex/CMakeModules/PlatformConfigLINUX.cmake plex-home-theater-1.3.5-nopulse/plex/CMakeModules/PlatformConfigLINUX.cmake
index bd8852d..730ee97 100644
--- plex-home-theater-1.3.5/plex/CMakeModules/PlatformConfigLINUX.cmake
+++ plex-home-theater-1.3.5-nopulse/plex/CMakeModules/PlatformConfigLINUX.cmake
@@ -91,10 +91,7 @@ if(CMAKE_USE_PTHREADS_INIT)
   set(HAVE_LIBPTHREAD 1)
 endif()
 
-plex_find_package(PulseAudio 0 1)
-if(HAVE_LIBPULSEAUDIO)
-  set(HAVE_LIBPULSE 1)
-endif()
+set(HAVE_LIBPULSE 0)
 plex_find_package(Alsa 0 1)
 
 plex_find_package(LibUSB 0 1)
